### PR TITLE
VEN-38 - adding circleci config.yml and a dummy test

### PR DESCRIPTION
**Pull request by shaulzorea**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      - run:
+          name: Authenticate with registry
+          command: |
+            echo "@melio:registry=https://npm.pkg.github.com/melio" > ~/.npmrc
+            echo "//npm.pkg.github.com/:_authToken=${GITHUB_TOKEN}" >> ~/.npmrc
+
+      - restore_cache:
+          key: v2-sam-{{ checksum "package-lock.json" }}
+
+      - run:
```
Comment (shaikegross) [inline .circleci/config.yml:54]: I think lines 54-70 can be removed.

When calling sam build, a custom runtime build will be triggered, this build is defined in [Makefile](https://github.com/melio/vendor-directory-api/blob/main/Makefile).

The `cp package-lokc.json` `npm i` `tsc` and `npm prune --production` will run from there

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "Api to access vendor directory V1.0 features",
   "main": "out/index.js",
   "scripts": {
-    "test": "jest",
+    "test:unit": "NODE_ENV=test TEST_MODE=unit mocha -r ts-node/register src/**/__tests__/*.test.ts --colors",
```
Comment (shaikegross) [inline package.json:7]: I think we should try and use jest, It came to my attention that other teams in Melio use jest. what do you think? not a deal breaker for me if you prefer to move ahead with mocha

**Review by shaikegross — CHANGES_REQUESTED** [review]
Please remove the steps that will happen as part of our custom build (more details in comments)

File: .circleci/config.yml (hunk 1 above)
Comment (shaulzorea) [reply .circleci/config.yml:54]: 👍🏼

File: package.json (hunk 2 above)
Comment (shaulzorea) [reply package.json:7]: i see both have large communities behind them - i don't mind switching

**Review by shaikegross — APPROVED** [review]

**Pull request merged**